server/frontend/package.json

```diff
@@ -1,57 +1,59 @@
 {
-    "name": "fuzzmanager",
```

Comment:
Don't change whitespace, please.

Reply:
Alright, the changes were introduced after running the format script: `"format": "prettier --write ./"`. We'll run it only on the files we worked on instead of the entire codebase.

Reply:
I suggest we leave the white spaces as they are since they were introduced by running the format script: `"format": "prettier --write ./"`.  It will help to  maintain consistency , and prevent potential conflicts going forward.

Your call though!